### DISCUSSION TITLE
MCU Shutdown: timer too close

### DISCUSSION THREAD

**Discussion by jeffalanmcclain** [xblax/flashforge_ad5m_klipper_mod#245] [Development]
This is *really* getting annoying.  I've now had 3 crashes in 7 prints, and I'm losing my mind after a long print halts 5 hours in.  I REALLY need some relief here.  The printer is GREAT for what it is, but this is intolerable.  Any word on when the next release or work on this MOD is?  Ideas on what to try to do to fix this?  I've already disabled Fluidd and Mainsail startup on the printer and am now running those remotely on a raspberry pi connection through moonraker to try to reduce load and over-head on the printer.  Unfortunately, unless I install a camera plugged into the rpi, the timelapse screen grabs still operate on the printer (through moonraker), but maybe (??) the ffmpeg video creation is on the rpi from fluidd (??? I'm not certain of this on where that operation takes place).  The crashes never really happen at the same place/time on a given g-code upload print.  For example, I can try to reprint a crashed operation from the same g-code and it can finish.

**Comment by EBiryukow** [comment]
The same issue. Tried any firmware. Error occured randomly

**Comment by maoravi456** [comment]
Using Guppyscreen solved this for me..

**Comment by KaruroChori** [comment]
Issue tracked at https://github.com/xblax/flashforge_ad5m_klipper_mod/issues/278
I am closing this one
